### PR TITLE
DOC: Fix Error in pandas.Series.last

### PR DESCRIPTION
**Pull request by farhanreynaldo**
- [ ] closes #xxxx
- [ ] tests added / passed
- [x] passes `black pandas`
- [x] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

Related to #27977. 

output of `python scripts/validate_docstrings.py pandas.Series.last`:
```
################################################################################
################################## Validation ##################################
################################################################################
```

### CODE REVIEW

**Review by WillAyd — APPROVED** [review]
lgtm

File: pandas/core/generic.py [hunk 1]
```diff
@@ -8062,15 +8062,21 @@ def first(self: FrameOrSeries, offset) -> FrameOrSeries:
...
+        Select final periods of time series data based on a date offset.
+
+        When having a DataFrame with dates as index, this function can
+        select the last few rows based on a date offset.
 
         Parameters
         ----------
         offset : str, DateOffset, dateutil.relativedelta
+            The offset length of the data that will be selected. For instance,
+            '1M' will display all the rows having their index within the first month.
```
Comment (simonjayhawkins) [inline pandas/core/generic.py:8074]: Is this correct? instead you could use '3D` here and reference the example below.

Comment (farhanreynaldo) [reply pandas/core/generic.py]: It's a little bit inaccurate, I'll fix it and refer to the example below.

**Review by simonjayhawkins — APPROVED** [review]

**Comment by simonjayhawkins** [comment]
Thanks @farhanreynaldo

**Pull request merged**